src/pycode.l

```diff
@@ -302,6 +302,12 @@ TARGET            ({IDENTIFIER}|"("{TARGET_LIST}")"|"["{TARGET_LIST}"]"|{ATTRIBU
                                       endFontClass(yyscanner);
                                       BEGIN( FunctionDec );
                                     }
+      "async"{BB}"def"{BB}          {
+                                      startFontClass(yyscanner,"keyword");
+                                      codify(yyscanner,yytext);
+                                      endFontClass(yyscanner);
+                                      BEGIN( FunctionDec );
+                                    }
```

Comment:
Not sure if this rule is really necessary, as `async` is already in the "KEYWORD" definition and we need the rule for `def` only as we need to go to a different state of the code scanner, the `async` does not result in a different state. (Though it is not that bad).